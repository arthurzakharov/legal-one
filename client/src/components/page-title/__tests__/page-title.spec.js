import React from 'react';
import renderer from 'react-test-renderer';
import PageTitle from '../page-title.component';

describe('PageTitle: ', () => {
  test('render with passed prop', () => {
    const component = renderer.create(<PageTitle title="x" />).toJSON();
    expect(component).toMatchSnapshot();
  });
  test('render without passed prop', () => {
    const component = renderer.create(<PageTitle />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
