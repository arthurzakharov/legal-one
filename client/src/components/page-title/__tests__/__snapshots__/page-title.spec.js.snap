// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle:  render with passed prop 1`] = `
<div
  className="PageTitle"
>
  x
</div>
`;

exports[`PageTitle:  render without passed prop 1`] = `
<div
  className="PageTitle"
/>
`;
